third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -1101,23 +1054,24 @@ impl Loader {
         //   If the module under verification declares a friend which is also in the bundle (and
         //   positioned after this module in the bundle), we defer the loading of that module when
         //   it is the module's turn in the bundle.
-        let locked_cache = self.module_cache.read();
+
+        // FIXME: Is there concurrency issue?
```

Comment:
I think we are making it the responsibility of module_cache to be thread-safe (and even now, when moved out, the locking happens inside has_module), so we shouldn't have to worry about it here?

Reply:
Yes. And particularly given the current loader is not shared across thread we should be fine as long as there isn't recursive lock acquisition.